src/App.js

```diff
@@ -91,24 +91,25 @@ const App = () => {
 
   const onSpin = async () => {
     try {
-      console.log("Bet: ", betAmount);
-      const tempA = await symbolGenerator();
-      const tempB = await symbolGenerator();
-      const tempC = await symbolGenerator();
-
-      setValA(tempA);
-      setValB(tempB);
-      setValC(tempC);
-      setBetAmount('');
+      // console.log("Bet: ", betAmount);
+      // const tempA = await symbolGenerator();
+      // const tempB = await symbolGenerator();
+      // const tempC = await symbolGenerator();
+
+      // setValA(tempA);
+      // setValB(tempB);
+      // setValC(tempC);
+      // setBetAmount('');
       
-      if(tempA===tempB && tempB===tempC)
```

Comment:
why is this logic here? Aren't we doing this logic in smart contract?

Reply:
This logic is in smart contract, hence has been commented out (for the time being. Will be removed.) in this iteration.

Reply:
ah my bad, i thought it was commented before and you un commented